ballista/executor/src/executor.rs

```diff
@@ -103,12 +105,22 @@ impl Executor {
         concurrent_tasks: usize,
         execution_engine: Option<Arc<dyn ExecutionEngine>>,
     ) -> Self {
+        let scalar_functions = all_default_functions()
```

Comment:
since we have these here, maybe we can remove the code that manually registers functions in the execution_loop?

Reply:
Yes, we should no longer need that